packages/docs/src/components/GithubLinks.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Looks like we're using React 17 for this project? According to docs, you [don't have to import React](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports) if you aren't using hooks, so this could probably be removed

Reply:
(This might apply to other files in this PR.)

Reply:
I may be missing it, but where do you see that we are using react 17? In the yarn.lock I see the react versions listed as
```
react@16.14.0, react@^16.14.0:
  version "16.14.0"
  resolved "https://registry.yarnpkg.com/react/-/react-16.14.0.tgz#94d776ddd0aaa37da3eda8fc5b6b18a4c9a3114d"
  integrity sha512-0X2CImDkJGApiAlcf0ODKIneSwBPhqJawOa5wCtKbu7ZECrmS26NvtSILynQ66cgkT/RJ4LidJOc3bUESwmU8g==
  dependencies:
    loose-envify "^1.1.0"
    object-assign "^4.1.1"
    prop-types "^15.6.2"
```

Reply:
Also, when I try to remove `import React from 'react'` I see this eslint error
```
'React' must be in scope when using JSXeslint react/react-in-jsx-scope
```
Is that an eslint rule we should change for just the `docs` directory?

Reply:
My bad - thought I saw it in the package file, but it was for React _types_, not React. Disregard!